src/Model/Filter/FilterLocator.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Search\Model\Filter;
+
+use Cake\Core\App;
+use Cake\Utility\Inflector;
+use InvalidArgumentException;
+use Search\Manager;
+
+/**
+ * Filter Locator
+ */
+class FilterLocator implements FilterLocatorInterface
+{
+    /**
+     * Manager
+     *
+     * @var \Search\Manager $manager Manager
+     */
+    protected $manager;
+
+    /**
+     * Constructor
+     *
+     * @param \Search\Manager $manager Manager
+     */
+    public function __construct(Manager $manager)
+    {
+        $this->manager = $manager;
+    }
+
+    /**
+     * Loads a search filter.
+     *
+     * @param string $name Name of the field
+     * @param string $filter Filter name
+     * @param array $options Filter options.
+     * @return \Search\Model\Filter\Base
+     * @throws \InvalidArgumentException When no filter was found.
+     */
+    public function locate($name, $filter, array $options = [])
```

Comment:
It would better to use method name `get()` akin to core's `TableLocator::get()` and `WidgetLocator::get()`.